packages/shared/lib/core/layer-2/constants/isc-contract.constant.ts

```diff
@@ -0,0 +1 @@
+export const ISC_CONTRACT = '0x1074000000000000000000000000000000000000'
```

Comment:
This is the magic contract right? Would be slightly less ambiguous to have `ISC_MAGIC_CONTRACT_ADDRESS = ...`

Reply:
updated ✅